### PR TITLE
Desc

### PR DESCRIPTION
**Pull request by krisdo**
--added description component. 
--refactored superhost and during your stay component to be a children and they're both condtional
--more tests

### CODE REVIEW

File: client/client-test/Description.spec.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Description from '../components/Description';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+ 
+configure({ adapter: new Adapter() });
+ 
+describe('Renders Components', () => {
+  let component;
+  let props = {
```
Comment (cmonez) [inline client/client-test/Description.spec.js:10]: Interesting way to pass props. I just keep a sampleData file that I import into my test suite. But I like this way because you can see exactly what goes in.

File: client/client-test/DuringYourStay.spec.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+describe('Renders Components', () => {
+  let component;
+  let prop = 'Commodi perspiciatis ullam et quod ipsa est porro. Ratione maiores rerum soluta sed. Dolores adipisci rerum sed omnis distinctio et facere cupiditate. Ut deserunt quo non praesentium id. Excepturi itaque adipisci consectetur voluptatibus. Sequi consequatur voluptatum et dolor.'
+  beforeEach( () => {
+    component = shallow(<DuringYourStay duringYourStay={prop}/>)
+  });
+
+  test("should render initial layout", () => {
+    const component = shallow(<DuringYourStay duringYourStay={prop} />);
+    expect(component.getElements()).toMatchSnapshot();
```
Comment (cmonez) [inline client/client-test/DuringYourStay.spec.js:17]: toMatchSnapshot? What does that do in laymans terms

File: client/components/Description.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Item } from 'semantic-ui-react';
```
Comment (cmonez) [inline client/components/Description.jsx:2]: Ohh using semantic ui. Curious to see what this ends up looking like!

File: client/client-test/DuringYourStay.spec.js (hunk 2 above)
Comment (krisdo) [reply client/client-test/DuringYourStay.spec.js:17]: It takes a snapshot of what's being rendered and then it checks if all the elements are on there
